vendor/tcib/nova-libvirt-arbiterd/nova-libvirt-arbiterd.yaml

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 - 2021, Sean Mooney and the arbiterd contributors
+# SPDX-License-Identifier: Apache-2.0
+---
+# we need a base contaienr with libvirtd and the nova user defiend so that
+# file permssions for the nova.conf are correct so use nova-libvirt?
+tcib_from:  quay.io/tripleomastercentos9/openstack-nova-libvirt:86f3dae907057e946a93f187b256cb6c
+tcib_actions:
+  - user: root
+  - run: adduser -Urm -G libvirt,nova arbiterd
+  - run: mkdir -m 770 /opt/arbiterd
+  - run: dnf install -y python3-pip
+  - run: chown arbiterd:arbiterd /opt/arbiterd
+  - user: arbiterd
+  - run: mkdir -p -m 660 /opt/arbiterd/etc/nova
+  - run: mkdir -p -m 660 /opt/arbiterd/etc/arbiterd
```

Comment:
- run: mkdir -p -m 660 /opt/arbiterd/etc/nova
 - run: mkdir -p -m 660 /opt/arbiterd/etc/arbiterd
so i was over thinking this  orginally

i was trying to ensure there would not be a conflict between arbiterd installed form package and arbtierd installed from pip by creating additional folder under opt.


can you update this to just use /etc/nova and /etc/arbiterd.

/etc/nova probably is already created in the base container so you can likely drop it entirely

/etc/arbiterd is currently unused but in the future I expect to introduce a config file which will store things
like the localization of the nova.conf so that is not required on the command line.

likely I will keep the option on the commandline too but allow you too use it form the config instead.

Reply:
Fixed in latest commit